docs/kb/semgrep-ci/ci-vs-cli.md

```diff
@@ -0,0 +1,71 @@
+---
+description: How to align your scan results between CI and CLI and understand differences in behavior.
+tags:
+  - Semgrep CI
+  - Semgrep CLI
+---
+
+# Semgrep in CI vs CLI: align your scan results and understand differences
+
+When configuring Semgrep, it can be helpful to run it both using the command-line interface (CLI) and in continuous integration (CI) in order to review findings behavior.
+
+However, the two methods of running Semgrep have somewhat different behavior by default, so the findings may not be directly comparable. If you're seeing different findings with a CLI scan as compared to a scan in CI, here are some possible reasons.
+
+## Installation methods and versioning
+
+When comparing Semgrep scans in CI and CLI, please start by ensuring that you are running the same version of Semgrep on the CLI as in CI, and that it is [installed](/docs/getting-started/#installing-and-running-semgrep-locally) in the same way as it is in CI.
+
+If you use Semgrep's Docker image in CI and are running the CLI scan locally, the best options are:
```

Comment:
Just curious - are you referring to [semgrep-agent](https://hub.docker.com/r/returntocorp/semgrep-agent)? do we have any customers running the Docker image, which we have deprecated since more than last year?

Reply:
@s-santillan No, maybe I'm saying this in a confusing way, though. We recommend a Docker image in several places in the current docs, usually for CI, but it can be used for CLI too. E.g. in the sample GHA config, we have these lines:

```
container:
      # A Docker image with Semgrep installed. Do not change this.
      image: returntocorp/semgrep
```

and a Docker option is also listed in the tabs for https://semgrep.dev/docs/getting-started/#installing-and-running-semgrep-locally.